ddtrace/_logger.py

```diff
@@ -33,13 +38,14 @@ def configure_ddtrace_logger():
     """
     ddtrace_logger = logging.getLogger("ddtrace")
 
+    if not os.environ.get("DD_TRACE_LOG_FILE"):
+        ddtrace_logger.addHandler(logging.StreamHandler())
```

Comment:
should this be and and not or?

e.g. do we always configure a stream handler, and then optionally trigger a log handler if asked for?

Reply:
We always need to configure a stream handler, otherwise logs injection will break if we remove `DD_CALL_BASIC_CONFIG` since it needs an active handler set on startup. 

My reasoning here is to either have a `StreamHandler` or a `FileHandler`, depending on if `DD_TRACE_LOG_FILE` is set, but avoid having both at the `ddtrace` level. This way we don't want to send logs to stdout if users want it sent to a file, and vice versa.

Reply:
> My reasoning here is to either have a StreamHandler or a FileHandler, depending on if DD_TRACE_LOG_FILE is set, but avoid having both at the ddtrace level. This way we don't want to send logs to stdout if users want it sent to a file, and vice versa.

After offline discussion: we have decided to go with adding a `StreamHandler` by default regardless if `DD_TRACE_LOG_FILE` is specified, meaning that in cases where `DD_TRACE_LOG_FILE` is specified we will log to both stdout and log file. We want to avoid the above as it is potentially unclear to users where their stdout logs might have went if they added file logging as well in this scenario.